### ISSUE TITLE
in a GPU with 24G memory, for one tick training, how long will it take? thanks

### ISSUE THREAD

**Issue by spencer03172023** [Zyriix/GDD#4]
"args": [
                "--workers=4",
                "--freeze=1",
                "--backbone=1",
                "--ema=0.5",
                "--dropout=0",
                "--batch=16",  // 单卡可减小batch
                "--dlr=1e-4",
                "--lr=1e-4",
                "--cond=0",
                "--arch=ncsnpp",
                "--duration=10",
                "--outdir=training-runs-cond-cifar10-d2o-f",
                "--transfer=pretrained/edm-cifar10-32x32-cond-ve.pkl",
                "--data=datasets/cifar10-32x32.zip",
                "--fp16=True",
                "--loss-type=ns",
                "--r1-gamma=1e-4",
                "--diffaug=False",
                "--r1-type=r1",
                "--ada-target=0",
                "--use-gp=True",
                "--d-type=xl",
                "--seed=42",
                "--precond=d2o",
                "--loss=d2o",

**Comment by Zyriix** [comment]
Thanks for your interest in our work.
For ImageNet64, training on 5 million images can be completed in approximately 48 A100 GPU-hours.
On a single 4090 GPU (24GB), training on 1 million images is estimated to take around 10–15 hours.
I suggest adjusting the --snap and --tick parameters to train on a small number of images first, so you can observe the speed and evaluate accordingly.

Hope this helps!

**Comment by spencer03172023** [comment]
Thanks, for full samples of CIFAR-10-32X32 on 4090, one tick takes around 20 mins. Thanks for your work, I can understand your whole picture of doing this. I just try to understand the detail. But before that, if my pre-train model is ncsnpp with VPPrecond type, and I modified the original ncsnpp network to reduce its parameters. But it was used for 512x512 image generation, does one 4090 GPU support this training?

**Comment by Zyriix** [comment]
Actually, I have tried different preconditioned pretrained models, including v-preditcion and epsilon-prediction variants. After converting them to x0-prediction, they still converge quickly.

For higher-resolution images, I'm not sure whether it can run within the 24GB memory limit of a single 4090 GPU.

**Comment by spencer03172023** [comment]
Thanks. After modification, i will let you know the result.
